### PR TITLE
khalid/Hide smartTrader for Australia

### PR DESCRIPTION
**Pull request by khalidibrahimio**
- SmartTrader is hidden for Australia with a message displayed
- Readme changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ You will also need to install the following on your development machine:
 - Ruby, RubyGems
 - Sass (`sudo gem install sass`)
 - Node.js (10.14.2 or higher is recommended) and NPM (see <https://nodejs.org/en/download/package-manager/>)
-- Go to project root, then run `npm install`
+- Go to project root, then run `npm ci`
```
Comment (amirghadimi-fs) [inline README.md]: Did someone ask you for this change? If yes, I think we should create a separate card for this

File: README.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ In order to remove the created folders from your gh-pages, you can use either:
 - `grunt shell:remove_folder --keep --folder=br_branchname1,br_branchname2,...`: only keeps the specified folder(s) on your gh-pages and removes everything else. Just add the `--keep` flag.
 
 ### Preview on your local machine
-- To preview your changes locally, run `sudo grunt serve`
+- To preview your changes locally, run `grunt start`
```
Comment (amirghadimi-fs) [inline README.md]: `start` command is for the first run of the project, it's not necessary for every change.

File: README.md (hunk 1 above)
Comment (khalidibrahimio) [reply README.md]: Yeah it was Mustofa. But you're right, it's better with a separate card. Can you create a card for it?

File: README.md (hunk 2 above)
Comment (khalidibrahimio) [reply README.md]: Oh okay, will update it

File: src/javascript/_common/base/client_base.js [hunk 3]
```diff
@@ -169,6 +169,7 @@ const ClientBase = (() => {
                 +authorize.local_currencies[local_currency_config.currency].fractional_digits;
         }
         set('email',      authorize.email);
+        set('country',   authorize.country);
```
Comment (mustofa-binary) [inline src/javascript/_common/base/client_base.js]:
```suggestion
        set('country',    authorize.country);
```

File: README.md (hunk 2 above)
Comment (mustofa-binary) [reply README.md]: the common misconception is that most people thought grunt serve will get their change before they run the command, meanwhile grunt serve is previewing the change after it runs the command. most people get used to npm start concept, where it bulid and serve your app. 

moreover, serve is only serving js, an css changes , jsx or adding new files they should run grunt shell:compile_dev and grunt copy respectively.

let's make PR to revamp binary readme to mention these in detail @amirghadimi-fs @khalidxx . we can combine it as well in the PR to change npm i to npm ci

Comment (khalidibrahimio) [reply README.md]: I have a PR for binary-static [here](https://github.com/binary-com/binary-static/pull/6386). I'll make for SmartTrader as well

**Review by mustofa-binary — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (amirghadimi-fs) [reply README.md]: @Khalidxx Could you please create it yourself? note that now we have SmartTrader project to assign a card. Thanks.

**Pull request merged**